Utilities/DataLoading/EdFi.LoadTools/Engine/MappingFactories/ResourceToResourceMetadataMappingFactory.cs

```diff
@@ -16,7 +16,7 @@ public class ResourceToResourceMetadataMappingFactory : MetadataMappingFactoryBa
             IEnumerable<XmlModelMetadata> xmlMetadata,
             IEnumerable<JsonModelMetadata> jsonMetadata,
             IEnumerable<IMetadataMapper> mappingStrategies)
-            : base(xmlMetadata, jsonMetadata, mappingStrategies) { }
```

Comment:
Why didn't this constructor change to List to match the other changes?

Reply:
defering for ODS-4721